boards/arm/particle_argon_nrf52/particle_argon_nrf52.dts

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright (c) 2018 Matthias Boesl
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <nordic/nrf52840_qiaa.dtsi>
+
+/ {
+	model = "particle argon nrf52";
+	compatible = "particle,particle_argon", "nordic,nrf52840-qiaa",
+		     "nordic,nrf52840";
+
+	chosen {
+		zephyr,console = &uart0;
```

Comment:
The only exposed UART with assigned pins does tx/rx on P0.6 and P0.7.  You've got that on `&uart1` below, which is fine, but then the chosen uarts should be `&uart1`, unless console should go to the WiFi coprocessor.